R/zeroPad.r

```diff
@@ -12,15 +12,23 @@
 #' correctPCode <- zeroPad(pCode,5)
 #' pCodes <- c('100','1000','0','12345','1565465465465465')
 #' correctPCodes <- zeroPad(pCodes,5)
+#' pCodeNA <- c(1,2,NA)
+#' padPCodeNA <- zeroPad(pCodeNA,4)
 zeroPad <- function(x,padTo){
   if(padTo <= 1) return(x)
+  #if(is.na(x) == TRUE) return(x)
   numDigits <- nchar(x)
   padding <- padTo-numDigits
-  padingZeros <- vapply(
+  padNA <- which(is.na(padding) == TRUE)
```

Comment:
We never need to use:
```r
blah == TRUE
```
If `blah` is a logical. Just say:
```
padNA <- which(is.na(padding))
```

Or to be more concise:
```r
if(any(is.na(padding))){
  message("There were NAs")
}
```
But...all that being said, I don't think this is `message`-worthy. I think it's a function that the user can expect to put in an `NA` and get out an `NA` without an Error.